resources/sdk/clisdkclient/extensions/transform_data/transform_data.go

```diff
@@ -70,4 +85,47 @@ func handleParse(t *template.Template, err error) *template.Template {
 		logger.Fatalf("Error parsing: %v\n", err)
 	}
 	return t
-}
\ No newline at end of file
+}
+
+func pullRemoteFileToTmpDir(url string) error {
+	rawURL := getFullUrlToRawFile(url)
+	var cmd *exec.Cmd
+
+	// curl request to pull file to current directory
+	cmd = exec.Command("curl", "-LJO", rawURL)
```

Comment:
Best to use http.Get for downloading. You can also avoid writing a temp file using this method because the file contents would be in the resp.Body variable